tools/lan_choose_character_kart/lan_choose_character_kart.py

```diff
@@ -0,0 +1,276 @@
+import itertools
+import os
+import subprocess
+from dataclasses import dataclass
+
+from tools.gc_c_kit import devkit_tools
+from tools.lan_choose_character_kart import symbols
+
+
+def run_subprocess(arguments: list[str]) -> subprocess.CompletedProcess:
+    completed_process = subprocess.run(arguments, capture_output=True)
+    assert completed_process.returncode == 0, f'ERROR CODE: {completed_process.returncode}, {completed_process.stdout.decode()}\n{completed_process.stderr.decode()}'
+    return completed_process
+
+
+def prepare_symbols_inc_file(filename: str, equ_symbols: dict[str, str],
+                             global_symbols: list[str]) -> None:
+    with open(filename, 'w') as symbolsfile:
+        equ_symbol_lines = '\n'.join([f'.equ {sym}, {equ_symbols[sym]}'
+                                      for sym in equ_symbols]) + '\n'
+        global_symbol_lines = '\n'.join([f'.global {adr}' for adr in global_symbols]) + '\n'
+        symbolsfile.write(equ_symbol_lines + global_symbol_lines)
+
+
+def remove_files(files: tuple[str]) -> None:
+    for filetodelete in files:
+        if os.path.exists(filetodelete):
+            try:
+                os.remove(filetodelete)
+            except:
```

Comment:
[Ruff](https://docs.astral.sh/ruff/) flags this with `Do not use bare except (E722)` (see [`bare-except (E722)`](https://docs.astral.sh/ruff/rules/bare-except/)).

```suggestion
            except Exception:
```